test/Microsoft.ML.OnnxTransformTest/DnnImageFeaturizerTest.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.ImageAnalytics;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Tests
+{
+    public class DnnImageFeaturizerTests : TestDataPipeBase
+    {
+        private const int inputSize = 3 * 224 * 224;
+
+        private class TestData
+        {
+            [VectorType(inputSize)]
+            public float[] data_0;
+        }
+        private class TestDataSize
+        {
+            [VectorType(2)]
+            public float[] data_0;
+        }
+        private class TestDataXY
+        {
+            [VectorType(inputSize)]
+            public float[] A;
+        }
+        private class TestDataDifferntType
+        {
+            [VectorType(inputSize)]
+            public string[] data_0;
+        }
+
+        private float[] getSampleArrayData()
+        {
+            var samplevector = new float[inputSize];
+            for (int i = 0; i < inputSize; i++)
+                samplevector[i] = (i / (inputSize * 1.01f));
+            return samplevector;
+        }
+
+        public DnnImageFeaturizerTests(ITestOutputHelper helper) : base(helper)
+        {
+        }
+
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))]
+        void TestDnnImageFeaturizer()
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Can't IOS run this featurizer?

Reply:
No, currently the Sonoma library we use to run ONNX is Windows only.

Reply:
This needs to be documented down in this `Estimator`'s doc string.